java-security/src/main/java/com/sap/cloud/security/token/validation/validators/XsuaaJwtSignatureValidator.java

```diff
@@ -1,18 +1,25 @@
 package com.sap.cloud.security.token.validation.validators;
 
 import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.config.ServiceConstants;
 import com.sap.cloud.security.token.Token;
 import com.sap.cloud.security.xsuaa.client.OAuth2ServiceException;
+import com.sap.cloud.security.xsuaa.http.HttpHeaders;
 
 import java.net.URI;
 import java.security.NoSuchAlgorithmException;
 import java.security.PublicKey;
 import java.security.spec.InvalidKeySpecException;
+import java.util.Collections;
+import java.util.Map;
 
 import static com.sap.cloud.security.token.validation.validators.JsonWebKeyConstants.*;
 
+/**
+ * Jwt Signature validator for Access tokens issued by Xsuaa service
+ */
 class XsuaaJwtSignatureValidator extends JwtSignatureValidator {
-    public static final String FALLBACK_KEY = "verificationkey";
+    public static final String FALLBACK_KEY = ServiceConstants.XSUAA.VERIFICATION_KEY;
```

Comment:
is it really necessary to store it under a new variable?

Reply:
Not really, thanks!